src/connectors/migu-music.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+Connector.playerSelector = '.mod-player';
+
+Connector.trackSelector = '.song a';
+
+Connector.artistSelector = '.singer a';
+
+Connector.albumSelector = '.album a';
+
+Connector.playButtonSelector = '.cf-player-play';
+
+Connector.timeInfoSelector = '.current-time';
+
+Connector.trackArtSelector = '.song-cover img';
+
+Connector.getTrackArt = () => {
+	const element = Util.queryElements(Connector.trackArtSelector);
+	if (!element) {
+		return null;
+	}
```

Comment:
Why do you override `getTrackArt`? When the `trackArtSelector` property is defined, the default implementation will get a track art URL from an element matching the selector.